utils/compare/search.py

```diff
@@ -0,0 +1,34 @@
+from elasticsearch import Elasticsearch
+from fastapi.encoders import jsonable_encoder
+import os
+# elasticsearch constants
+ES_URL = 'search-ocp-qe-perf-scale-test-elk-hcm7wtsqpxy7xogbu72bor4uve.us-east-1.es.amazonaws.com'
```

Comment:
Can we keep this as ENV too? Just don't wanted to get it exposed to the outside world.

Reply:
took these variables out and just using the ES_SERVER variable like the rest of kube-burner/other workloads export